plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/AvroSchemaConverter.java

```diff
@@ -208,7 +210,7 @@ private Optional<Type> convertRecord(Schema schema)
                 case IGNORE:
                     return Optional.empty();
                 case ADD_DUMMY:
-                    return Optional.of(RowType.from(ImmutableList.of(new RowType.Field(Optional.of(DUMMY_FIELD_NAME), BooleanType.BOOLEAN))));
+                    return Optional.of(DUMMY_ROW_TYPE);
```

Comment:
so it's now a RowType column called `dummy` with a single boolean field called `$dummy`?

i.e. `dummy ROW(dummy BOOLEAN)`?

Reply:
Yes: if there was an empty avro subrecord called `foo` then the schema will be `foo ROW("$dummy" BOOLEAN)`. I thought it would be more clear to prefix the dummy field with `$`. wdyt?